packages/clerk-js/src/core/clerk.ts

```diff
@@ -873,6 +876,23 @@ export default class Clerk implements ClerkInterface {
         buildURL({ base: displayConfig.signUpUrl, hashPath: '/continue' }, { stringify: true }),
     );
 
+    const navigateToNextStepSignUp = () => {
+      if (su.unverifiedFields.length === 0 || su.missingFields.length !== 0) {
```

Comment:
🔧 We shouldn't care about unverified fields here. If we have missing fields, we need to prompt the user to complete them, i.e. navigate to continue sign up.

Reply:
```suggestion
      if (su.missingFields.length) {
```